website/pages/downloads/style.css

```diff
@@ -1,4 +1,15 @@
 #p-downloads {
   margin-top: 72px;
   margin-bottom: 72px;
+
+  /** DEBT - temporary override until product download page is rolled out */
```

Comment:
Added a note on this to [the Nomad task on the Downloads rollout board](https://app.asana.com/0/1100423001970639/1199637416872535/f) - I think it's great we're meeting the contrast reqs, this feels like a good compromise to me ❇️